cs130_back/app.go

```diff
@@ -78,6 +78,10 @@ func (a *App) initializeRoutes() {
 	groupRoutes := routes.PathPrefix("/group").Subrouter()
 	groupRoutes.HandleFunc("/create", a.handleRequest(handlers.CreateGroup)).Methods("POST")
 
+	groupRoutes.HandleFunc("", a.handleRequest(handlers.GetGroup)).Methods("GET")
+	groupRoutes.HandleFunc("/update", a.handleRequest(handlers.UpdateGroup)).Methods("PUT")
+	groupRoutes.HandleFunc("/delete", a.handleRequest(handlers.DeleteGroup)).Methods("DELETE")
+
```

Comment:
Remember to make a path here for GetUserGroups. Lgtm otherwise, this should be merged in after the other PR is.